root/static/scripts/guess-case/modes.js

```diff
@@ -25,37 +27,42 @@ import {
  * warp     2011-02-01  added da, de, di, fe, fi, ina, inna
  * dpmittal 2016-12-20  added Turkish lowercase words
  */
-const LOWER_CASE_WORDS = /^(a|an|and|as|at|but|by|da|de|di|fe|fi|for|in|ina|inna|n|nor|o|of|on|or|tha|the|to|ve|ile|ya|veya|yahut|ki|mı|mi|mu|mü|mısın|musun|mudur|mıdır|midir|miyim|misin|misiniz|mısınız|müsün|müyüm|müsünüz|mıyım|muyum|musunuz|miyiz|mıyız|muyuz|müyüz|müdür)$/;
+const LOWER_CASE_WORDS = /^(a|aka|an|and|as|at|but|by|da|de|di|fe|fi|for|in|ina|inna|n|nor|o|of|on|or|tha|the|to)$/;
```

Comment:
This will mean it gets capitalized at the start / end of titles, which is probably not what we want (it should probably never be "Aka"). Should change it to match the way the other cleanup is done.

Reply:
Never? :clown_face: 
* [Gözyaşlarım Aka Aka](https://musicbrainz.org/recording/2610ec53-e9a5-40e1-8a64-b376c1f53ed5)
* [Aka Hip Aka Hop](https://musicbrainz.org/recording/efaaac5a-a96d-4c40-bec3-475158e085f1)